src/register/tests/RegistrationPage.test.jsx

```diff
@@ -627,6 +628,37 @@ describe('RegistrationPage', () => {
       mount(reduxWrapper(<IntlRegistrationPage {...props} />));
       expect(analytics.sendPageEvent).toHaveBeenCalledWith('login_and_registration', 'register');
     });
+
+    it('should select country based on IP address', () => {
```

Comment:
nit: rename the test to `Should update state if countryCode is present in context.`